code/game/objects/items/puzzle_pieces.dm

```diff
@@ -65,6 +65,10 @@
 	desc = "A key belonging to a once peaceful scholar, brought to death and ruin through means of violence by savage outsider."
 	puzzle_id = "priestkey"
 
+/obj/item/keycard/gatedrop/lavacanyon
```

Comment:
Add the key to the blood drunk's crusher loot pool as well. Megafauna drop different things when killed with a crusher.